inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -345,8 +345,7 @@ def prepare_keywords(obj, eng):
         # TODO: differentiate between curated and gueesed keywords
         keywords.append(
             {
-                'classification_scheme': '',
-                'keyword': keyword['label'],
+                'value': keyword['label'],
```

Comment:
there should be a 'schema' key, with probably 'INSPIRE' value (according to the jsonschema: https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/records/hep.json#L587